src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/importer/report/ObfuscatingReportImporterTest.kt

```diff
@@ -0,0 +1,50 @@
+package uk.gov.justice.digital.hmpps.pecs.jpc.importer.report
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.springframework.boot.test.mock.mockito.MockBean
+import org.springframework.context.annotation.Import
+import org.springframework.test.context.ActiveProfiles
+import org.springframework.test.context.junit.jupiter.SpringExtension
+import uk.gov.justice.digital.hmpps.pecs.jpc.TestConfig
+import uk.gov.justice.digital.hmpps.pecs.jpc.config.ReportingProvider
+import uk.gov.justice.digital.hmpps.pecs.jpc.service.MonitoringService
+import java.time.LocalDate
+
+@ExtendWith(SpringExtension::class)
+@Import(TestConfig::class)
+@ActiveProfiles("test")
+internal class ObfuscatingReportImporterTest() {
+
+  @MockBean
+  private lateinit var monitoringService: MonitoringService
+
+  @Test
+  fun `personal identifiable information is obfuscated`() {
+    val prisonNumber = "PRISON1"
+    val firstName = "Fred"
+    val lastName = "Blogs"
+    val ethnicity = "European"
+
+    val obfuscatedPerson = ObfuscatingReportImporter(
+      personProvider(
+        prisonNumber = prisonNumber,
+        firstName = firstName,
+        lastName = lastName,
+        ethnicity = ethnicity
+      ),
+      monitoringService
+    ).importPeopleOn(LocalDate.now()).toList().first()
+
+    assertThat(obfuscatedPerson.dateOfBirth).isEqualTo(LocalDate.of(1800, 1, 1))
```

Comment:
So a little hard to test this one.  Other than checking things are not the same I do not know what the obfuscated values will be as they are random.